README.md

```diff
@@ -1,6 +1,17 @@
 Angular 1 Samples
 ======
 
+## Note
+This branch of Angular sample is not compatible with our back end samples listed in [Using a different back end](#using-a-different-back-end)
+To use Angular sample with these back end samples, please checkout [Version 1.10](#https://github.com/okta/samples-js-angular-1/tree/1.10)
```

Comment:
Lets add an installation script here:
```bash
npm install @okta/samples-js-angular-1@1.10.0
```

Reply:
Also, we should split this into its own section. I'd probably fill this `README` with all info related to `2.x`, then link to a deprecated `README` file with instructions on running `1.x`

Reply:
Agreed. 
Have updated the README to include stuff only relevant to 2.x
Added a link to deprecated README with instructions running 1.x